client/devdocs/style.scss

```diff
@@ -1,91 +1,489 @@
+		linear-gradient( rgba( $gray-dark, 0.025 ) 1px, transparent 1px ),
+		linear-gradient( 90deg, rgba( $gray-dark, 0.025 ) 1px, transparent 1px );
+		background-size: 32px 32px, 32px 32px, 8px 8px, 8px 8px;
+		background-position: -1px -1px, -1px -1px, -1px -1px, -1px -1px;
+
+		// A wrapper inside a wrapper to help reliably center
+		// component and block examples.
+		.docs-example__wrapper-content-centering {
+			display: table;
+			margin: auto;
+			min-width: ( $devdocs-max-width / 2 );
+		}
+
+		// Some components use a card and the extra margin
+		// looks strange.
+		.card {
+			margin-bottom: 0;
+		}
+	}
+
+	// Styles specific to the single component view
+	.main.devdocs.is-single & {
+		.docs-example__wrapper-header-title {
+			padding: 16px 24px;
+		}
+	}
+
+	// How examples look in the lists of components view
+	.main.devdocs.is-list & {
+		break-inside: avoid;
+
+		.docs-example__wrapper-header-title a {
+			padding: 16px 24px;
+			display: block;
+		}
+
+		.docs-example__wrapper-content {
+			padding: 32px;
+			overflow: hidden;
+		}
+
+		// When we only have room for one column we
+		// set the max-width to line up with search
+		@include breakpoint( '<1280px' ) {
+			max-width: $devdocs-max-width;
+		}
+
+		// At really wide screens, we switch to a grid layout
+		// width fixed height examples and make the entire
+		// example clickable.
+		@media ( min-width: 2000px ) {
+			padding-top: 60px;
+
+			.docs-example__wrapper-header-title a {
+				position: absolute;
+					top: 0;
+					right: 0;
+					bottom: 0;
+					left: 0;
+			}
+
+			.docs-example__wrapper-content {
+				height: 140px;
+				overflow: hidden;
+				pointer-events: none;
+			}
+
+			&:hover {
+				border-color: $blue-light;
+
+				.docs-example__wrapper-content {
+					opacity: 0.7;
+				}
+			}
+		}
+	}
 }
 
-.devdocs__result-snippet {
+
+// README.md Viewer
+.docs-example__readme-viewer {
+	break-inside: avoid;
+
+	p, li {
+		font-size: 1.5rem;
+	}
+
 	p {
-		margin-bottom: 0;
+		margin-bottom: 0.5rem;
+	}
+
+	table {
+		border-collapse: collapse;
+		border-spacing: 0;
+
+		th, td {
+			border-bottom: 1px solid transparentize( $gray-lighten-20, 0.5 );
+			text-align: left;
+		}
+
+		th {
+			color: $blue-dark;
+			font-size: 0.75em;
+			padding: 0.25em 3em 0.25em 0;
+		}
+
+		td {
+			font-size: 0.83em;
+			padding: 1.17em 3em 1.17em 0;
+		}
 	}
+
+	h1, h2, h3, h4, h5, h6 {
+		display: block;
+		margin-bottom: 0.5em;
+		margin-left: 0;
+	}
+
+	h1 {
+		font-size: 1.33em;
+	}
+
+	h2 {
+		font-size: 1.17em;
+	}
+
+	h3 {
+    	font-size: 1em;
+	}
+
+	h4 {
+		font-size: 0.83em;
+	}
+
+	h5 {
+		font-size: 0.67em;
+	}
+
+	h6 {
+		font-size: 0.5em;
+	}
+}
+
+.docs-example__readme-viewer-not-available {
+	font-family: $sans;
+	text-align: center;
+	color: $gray-text-min;
+}
+
+
+// Some components/blocks use a button to toggle variants
+// This is really dumb right now.
+.docs__design-toggle,
+.design__toggle {
```

Comment:
I think we can remove `design__toggle` I can't find usages in code. Only exception would be folks are constructing class names like `'design_'+name`